assets/blocks/progress-bar.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * WordPress dependencies
+ */
+import { render } from '@wordpress/element';
+/**
+ * Internal dependencies
+ */
+import ProgressBar from '../shared/blocks/progress-bar';
+const barAttributes = {
+	style: {
+		...( window.sensei_quiz_progress.color && {
+			backgroundColor: window.sensei_quiz_progress.color,
+		} ),
+	},
+};
+
+const barWrapperAttributes = {
+	style: {
+		...( window.sensei_quiz_progress.backgroundColor && {
+			backgroundColor: window.sensei_quiz_progress.backgroundColor,
+		} ),
+		...( window.sensei_quiz_progress.height && {
+			height: parseInt( window.sensei_quiz_progress.height ),
+		} ),
+		...( window.sensei_quiz_progress.radius && {
+			borderRadius: parseInt( window.sensei_quiz_progress.radius ),
+		} ),
+	},
+};
+
+const wrapperAttributes = {
+	className: 'sensei-block-wrapper',
+};
+render(
+	<ProgressBar
+		lessonsCount={ window.sensei_quiz_progress.totalNumber }
```

Comment:
I think the `lessonsCount` should be renamed to something more generic. Maybe `totalCount`? Not sure...

Reply:
Will be covered in new PR